src/Core/Migration/V6_6/Migration1732821058UpdateCyprusVatValidationFormat.php

```diff
@@ -0,0 +1,39 @@
+<?php declare(strict_types=1);
+
+namespace Shopware\Core\Migration\V6_6;
+
+use Doctrine\DBAL\Connection;
+use Shopware\Core\Framework\DataAbstractionLayer\Doctrine\RetryableQuery;
+use Shopware\Core\Framework\Log\Package;
+use Shopware\Core\Framework\Migration\MigrationStep;
+
+/**
+ * @internal
+ */
+#[Package('core')]
+class Migration1732821058UpdateCyprusVatValidationFormat extends MigrationStep
+{
+    private const VAT_PATTERNS = [
+        'CY' => 'CY\d{8}[A-Z]'
+    ];
+
+    public function getCreationTimestamp(): int
+    {
+        return 1732821058;
+    }
+
+    public function update(Connection $connection): void
+    {
+        $update = new RetryableQuery(
```

Comment:
Please use the DAL for this type of change. In addition, a foreach is not necessary to change only one `vat_id_pattern`

Have a look at this one: https://github.com/shopware/shopware/blob/trunk/src/Core/Migration/V6_6/Migration1729843379FixBelgianVatIdPattern.php
Can you also create a changelog file and a testcase.
Like https://github.com/shopware/shopware/blob/trunk/tests/migration/Core/V6_6/Migration1729843379FixBelgianVatIdPatternTest.php